.travis.yml

```diff
@@ -8,6 +8,9 @@ install:
 notifications:
   slack:
     rooms:
-      - undeadbaby:$SLACK_KEY#random
+    - undeadbaby:WDGNd6FP2Uu61jNcbMaHLSIv#random
```

Comment:
The travis environment variables are documented here:

https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings

The page on which the variables are currently set is here:
https://travis-ci.org/operationzombie/apache-flask/settings (for the apache flask repo at least)
the docker stuff is definitely working

If you don't want to dig into this any more, I'll merge the PR, but if this had been something like an aws access key we'd be being billed for bitcoin mining right about now.